modules/codebuild/locals.tf

```diff
@@ -23,9 +23,10 @@ locals {
     if contains(values(lookup(project, "config", [])), var.git_repository_name)
   ]
 
-  get_codebuildspec_file = var.is_build_only ? "buildspec.ci.yaml" : "buildspec.yaml"
-  is_build_only          = var.is_build_only ? 1 : 0
-  is_deploy_only         = !var.is_build_only ? 1 : 0
+  get_codebuildspec_file  = var.is_build_only ? "buildspec.ci.yaml" : "buildspec.yaml"
+  get_codebuildspec_logic = var.buildspec_logic != null ? var.buildspec_logic : local.get_codebuildspec_file
```

Comment:
I'm not sure you need to change the logic.
the is_build_only variable is a boolean variable that has a default value. you never get null

Reply:
It just follows the logic you did in the multirepo module